### PR TITLE
Prepare hotfix release2.1.1

### PR DESCRIPTION
**Pull request by OsirisTerje**
HotFix for  issues #142 and issue #144 
On #142:  Separated output from the different projects so that we can release the adapter with Mono.Cecil  for version 3.5.    Reporter confirmed THIS issue was fixed.  Other reporter have another issue, which is still unknown, and doesnt seem related to the 3.5 issue, but seems related to Mono.Cecil. 
On #144:  Added same fix as for the NUnit3 adapter so that  MOno.Cecil doesnt crash when there is no pdb for an assembly

### CODE REVIEW

File: build.cake [hunk 1]
```diff
@@ -9,7 +9,7 @@ var configuration = Argument("configuration", "Debug");
 // SET PACKAGE VERSION
 //////////////////////////////////////////////////////////////////////
 
-var version = "2.1.0";
+var version = "2.1.1-b2";
```
Comment (rprouse) [inline build.cake]: Why the b2 suffix? Is this intended to be a beta or pre-release?

File: src/NUnitTestAdapterInstall/NUnitTestAdapterInstall.csproj [hunk 2]
```diff
@@ -30,7 +30,7 @@
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
-    <OutputPath>..\..\bin\Debug\</OutputPath>
+    <OutputPath>bin\Debug\</OutputPath>
```
Comment (rprouse) [inline src/NUnitTestAdapterInstall/NUnitTestAdapterInstall.csproj]: Why the change to the output directory? Doesn't this mess up the packaging?

File: build.cake (hunk 1 above)
Comment (OsirisTerje) [inline build.cake]: yes, i just used that for the version I uploaded to the reporter to check out.  I will remove this and make it 2.1.1 for the release

File: src/NUnitTestAdapterInstall/NUnitTestAdapterInstall.csproj (hunk 2 above)
Comment (OsirisTerje) [inline src/NUnitTestAdapterInstall/NUnitTestAdapterInstall.csproj]: The cake script picks the vsix from this directory. I just wanted to make sure I didn't inadvertently get any wrong dll's due to using a common output directory, because that is what happened.  So, I decided to just have separate output directories for each project, and pick from those.

**Comment by OsirisTerje** [comment]
The nuspec was correct with respect to directories, but what happened was that the test project copied its Mono dll's over to the same output directory as the adapter, and thus overwrote the 3.5 dll's with its own 4.5 versions.  So I just changed the output directory of the test project to avoid this.
